PennMobile/Events/EventsViewController.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  EventsViewController.swift
+//  PennMobile
+//
+//  Created by Samantha Su on 10/1/21.
+//  Copyright © 2021 PennLabs. All rights reserved.
+//
+
+import Foundation
+import SwiftSoup
+
+class EventsTableViewController: GenericTableViewController, IndicatorEnabled{
```

Comment:
Similar to the previous comment, lets rename this to `PennEventsTableViewController`